src/tabs/pill.scss

```diff
@@ -6,98 +6,54 @@
 
 @mixin iui-tabs-pill {
   @include iui-tabs;
-  display: flex;
   flex-flow: row;
   justify-content: space-evenly;
-  margin: $iui-baseline/2 0;
+  border-radius: $iui-border-radius;
+  gap: 2px;
+  @include themed {
+    background-color: t(iui-color-background-2);
+    border: 2px solid t(iui-color-background-2);
+  }
 
   > li {
-    margin: 0;
-    text-align: center;
     flex: 1 1 0;
 
-    > a {
-      line-height: $iui-line-height;
-      padding: ($iui-line-height * 0.5) 0;
-
-      @include themed {
-        background-color: t(iui-color-background-1);
-      }
+    > button {
+      height: $iui-baseline * 4;
+      background-color: transparent;
+      border-radius: $iui-border-radius;
+      width: 100%;
+      justify-content: center;
 
-      > svg {
+      &:enabled:hover {
         @include themed {
-          fill: t(iui-icons-color-actionable);
+          background-color: rgba(
+            t(iui-color-foreground-primary-rgb),
+            t(iui-opacity-6)
+          ) !important;
         }
-        transition: fill $iui-speed-fast ease-out;
-        width: $iui-icons-default;
-        height: $iui-icons-default;
       }
 
-      &::after {
-        bottom: $iui-xxs;
-        left: $iui-xxs;
-        right: $iui-xxs;
-      }
-
-      &:hover {
+      &.iui-active {
         @include themed {
-          color: t(iui-color-foreground-primary);
-        }
-
-        > svg {
-          @include themed {
-            fill: t(iui-icons-color-primary);
-          }
-          transition: fill $iui-speed-fast ease-out;
-        }
-      }
-    }
-
-    &.iui-tabs-active {
-      > a {
-        font-weight: $iui-font-weight-normal;
-
-        @include themed {
-          color: t(iui-color-foreground-primary);
-        }
-
-        &:hover {
-          @include themed {
-            color: t(iui-color-foreground-primary);
-          }
-        }
-
-        svg {
-          @include themed {
-            fill: t(iui-icons-color-primary);
-          }
+          background-color: rgba(
+            t(iui-color-foreground-primary-rgb),
+            t(iui-opacity-6)
+          );
+          box-shadow: inset 0 calc(#{$iui-xxs} * -1) 0 0;
```

Comment:
🤔

Reply:
`-#{$iui-xxs}` works too. Better change to this as it is easier to read.

Reply:
also, `xxs` is 2px, and we are using the value directly everywhere else in this PR

Reply:
Good suggestion, updated.

Reply:
Does `-$iui-xxs` not work? The `#{}` syntax is usually not necessary for numeric operations like `-`.

Reply:
`-$iui-xxs` appears to work on line 53 of `borderless.scss`, but does not work on line 46 of `pill.scss`.

Line 46 & results:
`box-shadow: inset 0 -$iui-xxs 0 0;`
![Screen Shot 2021-06-16 at 3 13 22 PM](https://user-images.githubusercontent.com/849817/122278872-6727dc00-ceb5-11eb-9ac9-b68ec500ee36.png)

Reply:
This appears to work: `box-shadow: inset 0 (-$iui-xxs) 0 0;`